op/model.py

```diff
@@ -259,10 +263,16 @@ class RelationNotFound(ModelError):
 
 
 class ModelBackend:
+
+    LEASE_REFRESH_PERIOD = datetime.timedelta(seconds=30)
```

Comment:
Is LEASE_CACHE_TIME easier to understand? (LEASE_CACHE_EXPIRY_TIME) it does start to get a bit wordy.
Not clearly better IMOl, just a brainstorm of words we could use.

Reply:
How about LEASE_RENEWAL_PERIOD?

I want to be explicit about it being a period of time, rather than a _point in time_. Usage of timedelta suggests that it's the case but the type is not exposed in the variable name.